### ISSUE TITLE
Preset Colors and Customer Color Schemes

### ISSUE THREAD

**Issue by thepeej-pwhite6** [hediet/vscode-drawio#125]
I'd really like to be able to configure draw.io that's embedded in Visual Code to be the same as the configuration I use outside of Visual Code.  I have setup Draw.io Desktop to use different pre-set colours (`presetColors`) and some custome styles (`customColorSchemes`).  I've attempted to add them in the Visual Code `settings.json` using the following.  Neither are recognized.  Is it possible to configure the extension to allow changing the pre-set colors and adding custom styles?  If not, can this be added to the backlog of enhancements?

```java
{
    "hediet.vscode-drawio.presetColors": [],
    "hediet.vscode-drawio.customColorSchemes": []
}
```

> :notebook: As an aside, when do you plan to update the embedded version of Draw.io in the extension.  I noticed the latest version uses a new style called "Sketch" whereas the previous version has "Comic".

**Comment by hediet** [comment]
Hey, sorry for the late response.

> As an aside, when do you plan to update the embedded version of Draw.io in the extension.

The current insiders build of this extension is already updated (you can find it in the marketplace). I'm waiting that enough people tested it so I can release it as stable.

> If not, can this be added to the backlog of enhancements?

Certainly. Also, I'm open for PRs here.

**Comment by AvroraPolnareff** [comment]
Can I help with adding these settings?

**Comment by hediet** [comment]
@AvroraPolnareff sure! 

You probably need to use [this](https://github.com/hediet/vscode-drawio/blob/master/src/DrawioInstance/DrawioTypes.ts#L82) property.

A value can be set [here](https://github.com/hediet/vscode-drawio/blob/master/src/DrawioWebviewInitializer/DrawioWebviewInitializer.ts#L64).

**Comment by hediet** [comment]
Implemented with #163.

**Comment by thepeej-pwhite6** [comment]
@hediet | @AvroraPolnareff - I big thank you to both of you.  I just downloaded the insiders version to find the update that enables  pre-set colors and customer color schemes to be defined.  All working nicely.  Agility at its best!⭐
